.github/workflows/docker.yml

```diff
@@ -151,7 +151,9 @@ jobs:
         run: |
           python3 -m appgate --spec-directory api_specs/${{ matrix.version }} dump-crd --version ${{ matrix.version }}
       - name: Move CRDs to chart templates
-        run: mv *-crd.yml k8s/crd/templates
+        run: |
+          mkdir k8s/crd/templates
```

Comment:
-p ?

Reply:
The parent directories are already there.